src/test/kotlin/com/exactpro/epfast/template/dsl/JavaProperty.kt

```diff
@@ -20,8 +20,10 @@ import kotlin.reflect.KProperty
 
 fun <T> javaProperty(getter: () -> T, setter: (T) -> Unit) = JavaProperty(getter, setter)
 
-class JavaProperty<T> internal constructor(private val getter: () -> T, private val setter: (T) -> Unit) {
-
+class JavaProperty<T> internal constructor(
+    private val getter: () -> T,
+    private val setter: (T) -> Unit
+) {
     operator fun getValue(thisRef: Any?, property: KProperty<*>): T = getter()
```

Comment:
NOT NOW: `throw RuntimeException("Attempt to read uninitialized property")` if `getter()` returns `null`.